code/style.css

```diff
@@ -1,4 +1,284 @@
-html {
-  background: #0025ff;
+*{
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+
+body{
+  width: 100%;
+  max-height: 100%;
+  background: rgb(34,193,195);
+  background: linear-gradient(0deg, rgba(34,193,195,0.7147233893557423) 0%, rgba(196,216,237,1) 58%);
+  display: flex;
+  flex-direction: column;
+  
+}
+
+
+
+header{
+display: flex;
+align-items: center;
+width: 100%;
+
+} 
+
+nav{
+width: 100vw;  
+height: 100px;
+}
+
+.logo{
+  font-family: 'Roboto', sans-serif;
+  font-size: 60px;
   color: #fff;
+  padding: 20px 0px 20px 60px;
+}
+
+.dropdown {
+  display: flex;
+  justify-content: flex-end;
+  align-content: flex-start; 
+  color: black;
+  padding: 14px;
+  line-height: 40px;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: rgb(197, 197, 197);
+  min-width: 10px;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  padding: 20px 10px;
+  border-radius: 10px;
+}
+
+.dropdown-list a {
+  color: #FFF;
+  font-size: 20px;
+  padding: 14px 13px;
+  text-transform: uppercase;
+  border-radius: 10px;
+  font-family: Arial, Helvetica, sans-serif;
+  }
+
+a:hover{
+    animation: blinker 2s linear infinite;
+  }
+
+
+@keyframes blinker {
+  50% {
+    opacity: 0;
+  }
+}
+
+
+.dropdown-content a:hover {
+  background-color: rgb(128, 51, 51);
+}
+
+.dropdown:hover .dropdown-content {
+  display:list-item;
+}
+
+.dropdown:hover .dropbtn {
+  background-color: rgb(255, 213, 198);
+}
+
+.container-1{
```

Comment:
It would be easier to follow your code if you have a more descriptive name of the containers.